osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -146,7 +146,8 @@ private Drawable[] createContent(int index, ScoreInfo score)
                 new OsuSpriteText
                 {
                     Text = $@"{score.MaxCombo:N0}x",
-                    Font = OsuFont.GetFont(size: text_size)
+                    Font = OsuFont.GetFont(size: text_size),
+                    Colour = score.MaxCombo == score.Beatmap.MaxCombo ? highAccuracyColour : Color4.White
```

Comment:
Better null handling is required (as evidenced by tests failing).

Reply:
woops, fixed